ios/RCTOneSignal/RCTOneSignalEventEmitter.m

```diff
@@ -361,4 +361,49 @@ + (void)sendEventWithName:(NSString *)name withBody:(NSDictionary *)body {
 RCT_EXPORT_METHOD(initInAppMessageClickHandlerParams) {
     //unimplemented in iOS
 }
+
+RCT_EXPORT_METHOD(addTriggers: (NSDictionary *)triggers) {
```

Comment:
Nit: what’s with the parameter spacing on the exported methods? Would suggest removing the spaces